### PR TITLE
Add extension support

### PR DESCRIPTION
**Pull request by Kidswiss**
## Summary

Removed the hardcoded `pg_repack`, as it will be handled by composition functions now.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: class/defaults.yml
Comment (Kidswiss) [inline class/defaults.yml:31]: Will be adjusted before merge.

Comment (zugao) [reply class/defaults.yml:31]: LGTM but please request again when the tag is adjusted

**Review by zugao — APPROVED** [review]
LGTM

**Pull request merged**
